api/src/main/java/io/opentelemetry/trace/Sampler.java

```diff
@@ -47,7 +47,7 @@ Decision shouldSample(
       TraceId traceId,
       SpanId spanId,
       String name,
-      List<Span> parentLinks);
+      List<SpanContext> parentLinks);
```

Comment:
Should we use List<Link>? That way we don't have to do any transformation in the Builder.

Reply:
You mean `List<Link>`? If so I think that makes more sense.

Reply:
Updated in https://github.com/open-telemetry/opentelemetry-java/pull/455/commits/99b3816292145a9501438ca3e63d9e6e575fcd88.